e2e-tests/pipeline_validation.sh

```diff
@@ -106,25 +106,25 @@ function setup() {
   rm -rf "${HOME_PATH}/fhir"
   rm -rf "${HOME_PATH}/${PARQUET_SUBDIR}/*.json"
   find "${HOME_PATH}/${PARQUET_SUBDIR}" -size 0 -delete
-  FHIR_SERVER_URL='http://localhost:8099/openmrs/ws/fhir2/R4'
-  SINK_SERVER='http://localhost:8098'
+  SOURCE_FHIR_SERVER_URL='http://localhost:8099/openmrs/ws/fhir2/R4'
```

Comment:
Should we change the default here _not_ to be OpenMRS based too?

Reply:
Done! Had to make a few changes in the file